internal/domain/target.go

```diff
@@ -3,11 +3,8 @@ package domain
 type TargetType uint
 
 const (
-	TargetTypeUnspecified TargetType = iota
-	TargetTypeWebhook
+	TargetTypeWebhook TargetType = iota
 	TargetTypeRequestResponse
-
-	TargetTypeStateCount
```

Comment:
why do we drop the unspecified?

Reply:
As the TargetType always has to be set, and in the proto there is a oneOf which is required.